includes/ui/class-ui.php

```diff
@@ -33,12 +33,20 @@ class UI {
 	 *
 	 * @var bool
 	 */
-	private $has_ui = false;
+	private $has_ui;
+
+	/**
+	 * An array of css variables to be enqueued with the styles.
+	 *
+	 * @var array
+	 */
+	private $css_variables;
```

Comment:
```suggestion
	private array $css_variables;
```

I think we can type properties on PHP 7.4 :)